packages/pc/src/behavior/brush-select.ts

```diff
@@ -170,16 +177,39 @@ export default {
         });
       });
     }
+    const selectedCombos = [];
+    if (this.includeCombos) {
+      graph.getCombos().forEach((combo) => {
+        if (!combo.isVisible()) return;
+        const bbox = combo.getBBox();
+        if (
+          bbox.centerX >= left &&
+          bbox.centerX <= right &&
+          bbox.centerY >= top &&
+          bbox.centerY <= bottom
+        ) {
+          if (shouldUpdate(combo, 'select')) {
+            selectedCombos.push(combo);
+            const model = combo.getModel();
+            selectedIds.push(model.id);
+            graph.setItemState(combo, state, true);
+          }
+        }
+      });
+    }
+
 
     this.selectedEdges = selectedEdges;
     this.selectedNodes = selectedNodes;
+    this.selectedCombos = selectedCombos;
     if (this.onSelect) {
-      this.onSelect(selectedNodes, selectedEdges);
+      this.onSelect(selectedNodes, selectedEdges, selectedCombos);
     }
     graph.emit('nodeselectchange', {
       selectedItems: {
         nodes: selectedNodes,
         edges: selectedEdges,
+        combos: selectedCombos,
```

Comment:
```suggestion
        combos: selectedCombos
```

Reply:
The existing code already had commas even on the last property, so I preferred to keep the style. If it's a style thing they'd probably as me to re-add those commas in the upstream PR, so let's just preserve the status quo.